webapp/components/reftest-analyzer.js

```diff
@@ -101,9 +104,11 @@ class ReftestAnalyzer extends LoadingState(PolymerElement) {
         </div>
```

Comment:
Optional: add tooltips to the two radio buttons to show their keyboard bindings.

Reply:
I'll avoid this for now as the binding behaviour is added in the page (body keydown), not the component.